helper-cli/src/main/kotlin/commands/ExportPathExcludesCommand.kt

```diff
@@ -75,7 +77,10 @@ internal class ExportPathExcludesCommand : CommandWithHelp() {
             .replaceConfig(repositoryConfigurationFile.readValue())
             .getRepositoryPathExcludes()
 
-        val globalPathExcludes = pathExcludesFile.readValue<RepositoryPathExcludes>()
+        @Suppress("TooGenericExceptionCaught")
```

Comment:
Isn't this another topic than the one mentioned in the commit message? This is not about writing to non-existing *output* files / dirs, but about not requiring this *input* file. So I'd prefer to either generalize the commit message, or split this out into a new commit.

In any case, a more elegant solution would be to explicitly test for the file existence:

        val globalPathExcludes = if (pathExcludesFile.isFile) {
            pathExcludesFile.readValue<RepositoryPathExcludes>()
        } else {
            mapOf()
        }

That way you also don't have to suppress the *detekt* finding.

Reply:
> Isn't this another topic than the one mentioned in the commit message? This is not about writing to non-existing _output_ files / dirs, but about not requiring this _input_ file.

This is not an _input_ file but an _output_ file.

Reply:
> This is not an _input_ file but an _output_ file,

I'm confused. You do `readValue<RepositoryPathExcludes>()`, so you *read* from that file. How can it then be an output file?